apps/stress-test/src/renderers/wc/btn/wcBasicButton.ts

```diff
@@ -2,13 +2,11 @@ import { DOMSelectorTreeComponentRenderer } from '../../../shared/vanilla/types'
 
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30689&commitId=3d7e6b986455f286921229650fba9bbbddfb19c0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Box</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Box.Box Shorthand Example.chromium.png|12| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

Reply:
@layershifter @ling1726 Not sure what's going on with the change here. Noise?